CONTRIBUTING-development.md

```diff
@@ -0,0 +1,150 @@
+# Coding style
```

Comment:
I think it's good to prefix this whole file with a disclaimer, saying something like:

> The existing codestyle in our codebase is not super consistent. There are many reasons for that, one common one is that our styleguide is different from Postgres its styleguide and some code is copied from postgres source code and slightly modified. The below rules are for new code. If you're changing existing code that uses a different style, use your best judgement to decide if you use the rules here or if you match the existing style.